server/views/pages/contacts/manage/contactConfirmation/confirmation.njk

```diff
@@ -77,6 +78,7 @@
                 <div class="govuk-tabs__panel--no-border govuk-!-static-padding-top-7 govuk-tabs__panel--hidden" id="linked-offenders"></div>
             </div>
 
+            <h2 class="govuk-heading-l" data-qa="confim-title-value-bottom">{{ isTheRightPersonTitle }}</h2>
```

Comment:
I think it should use the fieldset legend for this:
https://design-system.service.gov.uk/components/radios/

Reply:
fieldset legend added